### DISCUSSION TITLE
NPM Install Errors: Troubleshooting Win!

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#43] [Look at what I learned!]
Your Name: Dani Crosby
Have you searched the other help tickets? Yes

Describe your issue: npm test main.test.js stopped working. I left my computer on overnight and when I came back to my js-2 project, my NPM wasn't running correctly. None of my tests were showing and the below error kept showing when I refreshed. I tried a few things with no avail, but had a hunch to just checkout back to my main/master branch, run the NPM install again, then back to my attempt1 branch and run the npm test again AND THAT WORKED! My testing is back on track!

List any error messages that appear: 

    Just encountered an unexpected token
    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.
    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

**Comment by jackiebgregory** [comment]
I just had a very similar issue! Left it up overnight as well, and when I came back to it this evening, it DID work at first. I did another practice problem, ran the npm test, and it worked. Then I did another problem, ran it, and just ... didn't seem to do anything - it stopped at RUNS./main.test.js. So restarted my computer, followed these instructions, but... still nothing. So I will go fill out a Help Ticket, but thank you for getting me started with ideas to try!
